<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneWindow</class>
 <widget class="QMainWindow" name="SceneWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>771</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courant</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Дисперсия в скрещенных призмах</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="ScOpenGLWidget" name="sceneWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockSceneParams">
   <property name="maximumSize">
    <size>
     <width>405</width>
     <height>156</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockSceneParamsContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <item row="6" column="0">
      <widget class="QLabel" name="labelFirstPrismRotation">
       <property name="text">
        <string>Поворот</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="Line" name="separatorLayoutButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="labelFirstPrismAngle">
       <property name="text">
        <string>Угол при вершине</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="labelFirstPrism">
       <property name="text">
        <string>Первая призма</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QSpinBox" name="spinBoxFirstPrismRotation">
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="minimum">
        <number>-10</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spinBoxFirstPrismAngle">
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="layoutSetupControls">
       <item>
        <widget class="QPushButton" name="pushButtonSetupOnePrism">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Одна призма</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSetupTwoPrisms">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Две призмы</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBoxSecondPrismAnomalous">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Аномальная дисперсия</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockRefractiveIndexParams">
   <property name="maximumSize">
    <size>
     <width>405</width>
     <height>524287</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <item row="20" column="0">
      <widget class="QLabel" name="labelAbspSpectrumWidthMin">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>20</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="21" column="0" colspan="3">
      <widget class="QLabel" name="labelAbspSpectrumHeight">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Макс. коэффициент поглощения </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="22" column="0" colspan="3">
      <widget class="QSlider" name="sliderAbspSpectrumHeight">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>38</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="sliderPosition">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="3">
      <widget class="QSlider" name="sliderAbspSpectrumCenter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>380</number>
       </property>
       <property name="maximum">
        <number>780</number>
       </property>
       <property name="value">
        <number>580</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="labelAbspSpectrumCenterMid">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>580</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="23" column="2">
      <widget class="QLabel" name="labelAbspSpectrumHeightMax">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="labelAbspSpectrumCenterMin">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>380</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="20" column="2">
      <widget class="QLabel" name="labelAbspSpectrumWidthMax">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>120</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QLabel" name="labelAbspSpectrumWidthMid">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>70</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="19" column="0" colspan="3">
      <widget class="QSlider" name="sliderAbspSpectrumWidth">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>120</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1046</number>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="3">
      <widget class="QLabel" name="labelAbspSpectrumCenter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Положение линии поглощения (нм)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="18" column="0" colspan="3">
      <widget class="QLabel" name="labelAbspSpectrumWidth">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ширина линии поглощения (нм)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="23" column="0">
      <widget class="QLabel" name="labelAbspSpectrumHeightMin">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="23" column="1">
      <widget class="QLabel" name="labelAbspSpectrumHeightMid">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0.5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QLabel" name="labelAbspSpectrumCenterMax">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>780</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="ScPlotViewWidget" name="chartRefractiveIndex">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>170</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <layout class="QGridLayout" name="layoutPlotLabels">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="labelPlotRefr">
         <property name="text">
          <string>Показатель преломления</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="imageLabelPlotRefr">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/Images.qrc">:/gfx/green.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="imageLabelPlotAbsp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/Images.qrc">:/gfx/blue.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelPlotAbsp">
         <property name="text">
          <string>Коэффициент поглощения</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="24" column="0" colspan="3">
      <widget class="QPushButton" name="pushButtonToMenu">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>widgets/SceneWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ScPlotViewWidget</class>
   <extends>QGraphicsView</extends>
   <header>widgets/PlotViewWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/Images.qrc"/>
 </resources>
 <connections/>
</ui>
